File: python/6 kyu/272.py
# https://www.codewars.com/kata/5971b219d5db74843a000052/train/python

# Passed

from collections import Counter

def added_char(s1, s2):  
    s1_counter = Counter(s1)
    s2_counter = Counter(s2)
    
    for char in s2_counter:
        if (s2_counter[char] - s1_counter[char]) == 3:
            return char

output = added_char("hello","checlclo")
print(output)

File: nodejs/7 kyu/272.js
// https://www.codewars.com/kata/602db3215c22df000e8544f0/train/javascript

// Passed

function twoArePositive(a, b, c) {
  let positiveCount = 0;
  nums = [a, b, c];
  for (const num of nums) {
    if (num > 0) {
      positiveCount++;
    }
  }

  let result = positiveCount === 2;
  return result;
}

const output = twoArePositive(2, 4, -3);
console.log(output);